### PR TITLE
New rest api config.

### PR DESCRIPTION
**Pull request by gjersvik**
Moved mutch of the config to a common .env at the root of the project with a goal that this is what RestApi, Frontend, Docker and other system uses. So that we do not need to configure everything.

The RestApi uses a new ConfigService that helps with the limitations of the MS Configuration system.

The big change is that all the DB settings have been consolidated to a connection string. You can find it in azure or ask me.

### CODE REVIEW

**Review by Unicus-Tom — APPROVED** [review]

File: RestApi/Sepes.RestApi/Model/DatabaseConfig.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+namespace Sepes.RestApi.Model
+{
+
+    public class DatabaseConfig
+    {
+        public string connectionString {get;}
```
Comment (Kris2ffer) [inline RestApi/Sepes.RestApi/Model/DatabaseConfig.cs]: It seems inconsistent that DatabaseConfig has a string member with the connection string, and not an SqlConnection, while AzureConfig has an AzureCredentials member and not any string members.

**Review by Kris2ffer — CHANGES_REQUESTED** [review]
I think having a config service is a good idea. 
Having configurations for each service separated into models seams unnecessary, but I can see that if more configurations for new services are added over time, Config Service would get messy. Also, separation of ownership is good.

AppSettings.cs should be renamed.
SepesDB.cs in Models should be deleted.

File: RestApi/Sepes.RestApi/Model/DatabaseConfig.cs (hunk 1 above)
Comment (gjersvik) [reply RestApi/Sepes.RestApi/Model/DatabaseConfig.cs]: The reason for models per service is that makes sure that etch service get only what it needs and don't take what it wants. And for sepes there is not a lot of services that need configurations. So this is as bad as it gets.

ConfigService can become messy yes but it keeps everything else clean. And if it becomes to messy it only one place to refactor.

Will do the changes did not want to mess with more code than i needed.

On the DatabaseConfig its only a model to make DI happy but i did not end up using it. So could just be a string in the cconstukor. Will look into using SqlConnection. For me to use SqlConnection i need to make sure its only model as in setting its values do not have side effects. (Like connection to a database.) And that it keeps Config Servcie somewant testable.

I will look at SqlConnection and use that or just a string. And remove DatabaseConfig as its not needed outside of auto DI

Comment (gjersvik) [reply RestApi/Sepes.RestApi/Model/DatabaseConfig.cs]: SqlConnection do not work well as a config object as behavior with side effects. Where AzureCredentials do no do anything until you give to a Azure Client.

**Review by Kris2ffer — APPROVED** [review]

**Pull request merged**